src/lib/common/tag.h

```diff
@@ -130,7 +128,6 @@ extern std::string valueTag
   const std::string&  value,
   Format              format,
   bool                showComma           = false,
-  bool                isAssociation       = false,
   bool                valueIsNumberOrBool = false
 );
 
```

Comment:
The first and the third valueTag are identical, as far as I can see.
Does this complie?